api-specification.md

```diff
@@ -0,0 +1,652 @@
+# API Access
+
+The Flow Results specification is designed to be useful in both file-based and API-based environments. File-based usage is ideal for archiving and transferring data manually between systems. However, in many production situations, interoperable systems are likely to exchange data via HTTP-based API services. When implementations provide access to Flow Results Packages and Responses via an API, the following standardized endpoints, parameters, and envelope formats shall be used.
```

Comment:
I'd make the final sentence active voice, like "When implementations provide access to Flow Results Packages and Responses via an API, they should use the following standardized endpoints, parameters, and envelope formats."

Reply:
Sounds good.